src/applications/disability-benefits/686/components/SpouseMarriageTitle.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { getSpouseMarriageTitle } from '../helpers';
```

Comment:
might be cleaner to call this in the config and pass the string into the component.

Reply:
It needs the index of the current page, so I don't think I can call it in the config.

Reply:
I understand now. I was reading `SpouseMarriageTitle` and `getSpouseMarriageTitle` as the same thing.